utils/user_info.py

```diff
@@ -191,3 +198,27 @@ def send_forget_password_email(request, user):
         Thread(target=msg.send, args=()).start()
     except:
         logger.warning("There is an error when sending email to %s's mailbox" % username)
+
+def get_public_users():
+    return User.objects.filter(username__startswith = settings.ANONYMOUS_PREFIX)
```

Comment:
I suggest to use [count](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#count)